### PR TITLE
Fix /var/tmp/test_payload_file handling

### PR DESCRIPTION
**Pull request by kkaarreell**

### CODE REVIEW

File: functional/basic-attestation-with-unpriviledged-agent/test.sh [hunk 1]
```diff
@@ -22,6 +22,8 @@ rlJournalStart
             rlRun "limeUpdateConf cloud_verifier revocation_notifiers ''"
             rlRun "limeUpdateConf cloud_agent listen_notifications False"
         fi
+        # change /etc/keylime.conf permissions so that agent running as ${AGENT_USER} can access it
+        rlRun "chmod 644 /etc/keylime.conf"
```
Comment (sergio-correia) [inline functional/basic-attestation-with-unpriviledged-agent/test.sh:26]: Don't you need to undo this change in the cleanup step?

Comment (kkaarreell) [reply functional/basic-attestation-with-unpriviledged-agent/test.sh:26]: we call `limeBackupConfig` at above and `limeRestoreConfig` in the Clean up phase.

**Review by sergio-correia — APPROVED** [review]
